_includes/charts/tigera-operator/templates/tigera-operator/02-role-tigera-operator.yaml

```diff
@@ -125,42 +169,98 @@ rules:
     verbs:
       - create
 {{- end }}
-    # Permissions below this point are required for TSEE only.
+  # Permissions below this point are required for TSEE only.
```

Comment:
After chatting a bit with Casey, we should remove everything below here. There isn't really any benefit to including it for Calico installations. We're not including the Enterprise CRDs so switching to Enterprise without any other additions is not possible.  This will likely get out of date because there isn't anything keeping the Calico and Enterprise versions in sync either.